frontend/web/components/App.js

```diff
@@ -254,6 +258,17 @@ const App = class extends Component {
         this.setState({ totalApiCalls: res[0]?.data?.totals.total })
       })
     }
+    const subscriptionModel = OrganisationStore.getSubscriptionMeta()
```

Comment:
One thing that I'm concerned about here is that for each request the client makes to `get-subscription-metadata`, we are making a call to Chargebee's API. What that means is that: 

 a. This endpoint is not the most performant endpoint we have (as we're reliant on the performance of an external API). We should therefore ensure that we're not blocking anything waiting on the response from this endpoint. Maybe that's already happening here, but I'd like to confirm that. 
 b. For each request to this endpoint on our side, we trigger a request to Chargebee's API which has rate limits, etc. so we should be careful about abusing it. 

Here are the CB API limits. We are on the 'Rise' plan, as per the highlighted section. 

<img width="595" alt="Screenshot 2023-06-19 at 15 07 08" src="https://github.com/Flagsmith/flagsmith/assets/14089968/73698c57-ba2d-4a6d-aefc-d0e5cd0418f7">

Based on the above, is there anything we can do to optimise the requests from the frontend? Or, perhaps we should optimise this on the BE? Store some kind of additional cache of a customer's subscription information after a successful request from CB? 

I'm guessing that by adding this here, we're triggering the request on every successful load of the Flagsmith dashboard. I don't know where our current traffic is right now, but I don't imagine this number is beyond the realms of possibility. 

cc @kyle-ssg @dabeeeenster

Reply:
Can we just store this with a 24 hour TTL in our DB? I thought we were already storing this data in our database? Isn't that what `organisations_organisationsubscriptioninformationcache` is for?

Reply:
> Can we just store this with a 24 hour TTL in our DB?
Yes, I think that's what I would suggest. 

>I thought we were already storing this data in our database? Isn't that what organisations_organisationsubscriptioninformationcache is for?
Currently this table is only used for the sales dashboard. We could, however, expose this to the frontend for sure. We should make sure we implement the recurring task before we do that though.

Reply:
@matthewelwell do we have in place a job to execute the update of this data? That is what you are talking about when you say "recurring task" right?

Reply:
Yep, you can see the functionality [here](https://github.com/Flagsmith/flagsmith/blob/main/api/task_processor/models.py#L126).

Reply:
TODO: 

 * Use CB webhook to keep OrganisationSubscriptionInformationCache up to date with max_seats and max_api_calls
 * Update the get-subscription-metadata endpoint to use the OrganisationSubscriptionInformationCache object 
 * Update the current task which updates the OrganisationSubscriptionInformationCache models to ONLY update influx data

To clarify, we should continue using live influx data in the frontend dashboard.

Reply:
Done